src/views/transaction/components/fees-table/fees-table.view.jsx

```diff
@@ -47,14 +49,16 @@ function FeesTable({ l2Fee, estimatedWithdrawFee, token, preferredCurrency, fiat
       <div className={classes.feesTable}>
         <TransactionInfoRow
           title="Hermez fee"
-          subtitle={`${CurrencySymbol[preferredCurrency].symbol}${getL2FeeInFiat()}`}
+          subtitle={<FiatAmount amount={getL2FeeInFiat()} currency={preferredCurrency} />}
```

Comment:
This is still typed as `string` so we get this error:
![image](https://user-images.githubusercontent.com/15896177/137307629-f666965c-2537-4a4f-8ec7-014f4cb3ed4a.png)